apps/server/src/modules/video-conference/uc/video-conference-deprecated.uc.spec.ts

```diff
@@ -19,7 +19,8 @@ import { CalendarEventDto } from '@shared/infra/calendar/dto/calendar-event.dto'
 import { TeamsRepo, VideoConferenceRepo } from '@shared/repo';
 import { roleFactory, setupEntities, userDoFactory } from '@shared/testing';
 import { teamFactory } from '@shared/testing/factory/team.factory';
-import { AuthorizationService, LegacySchoolService, UserService } from '@src/modules';
+import { LegacySchoolService, UserService } from '@src/modules';
+import { AuthorizationReferenceService } from '@src/modules/authorization/domain/service';
```

Comment:
shouldn't this be behind a proper barrel-file on a higher level?

Reply:
No it is a import violation and should visible for all. It use a service that is not avaible outside the authorisation module.